IFC 4.3/SectionSolidHorizontal_1/UT_SectionSolidHorizontal_1.ifc

```diff
@@ -0,0 +1,90 @@
+#15 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#16 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-4, #17, #19);
+#17 = IFCAXIS2PLACEMENT3D(#18, $, $);
+#18 = IFCCARTESIANPOINT((0., 0., 0.));
+#19 = IFCDIRECTION((0., 1., 0.));
+#20 = IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body', 'Model', *, *, *, *, #16, $, .MODEL_VIEW., $);
+#21 = IFCARBITRARYCLOSEDPROFILEDEF(.AREA., $, #22);
+#22 = IFCPOLYLINE((#23, #24, #25, #26));
+#23 = IFCCARTESIANPOINT((-4., 5.E-1));
+#24 = IFCCARTESIANPOINT((-4., 0.));
+#25 = IFCCARTESIANPOINT((-12., 0.));
+#26 = IFCCARTESIANPOINT((-12., 5.E-1));
+#27 = IFCARBITRARYCLOSEDPROFILEDEF(.AREA., $, #28);
+#28 = IFCPOLYLINE((#29, #30, #31, #32, #33, #34, #29));
+#29 = IFCCARTESIANPOINT((-2.75, 5.E-1));
+#30 = IFCCARTESIANPOINT((-3.6, 1.1));
+#31 = IFCCARTESIANPOINT((-4.15, 1.));
+#32 = IFCCARTESIANPOINT((-6.85, 1.));
+#33 = IFCCARTESIANPOINT((-7.4, 1.1));
+#34 = IFCCARTESIANPOINT((-8.25, 5.E-1));
+#35 = IFCSITE('0zL93swtHAG9KGJuCT0x1e', #1, 'Default Site', $, $, #36, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#36 = IFCLOCALPLACEMENT($, #39);
+#37 = IFCDIRECTION((0., 0., 1.));
+#38 = IFCDIRECTION((1., 0., 0.));
+#39 = IFCAXIS2PLACEMENT3D(#18, #37, #38);
+#40 = IFCALIGNMENT('0Ax$L$fOP3jArc6r1IB9Y$', #1, 'example alignment', $, $, #41, $, #45, $);
+#41 = IFCLOCALPLACEMENT(#36, #44);
+#42 = IFCDIRECTION((0., 0., 1.));
+#43 = IFCDIRECTION((1., 0., 0.));
+#44 = IFCAXIS2PLACEMENT3D(#18, #42, #43);
+#45 = IFCALIGNMENTCURVE(#46, $, $);
+#46 = IFCALIGNMENT2DHORIZONTAL($, (#47, #50, #53));
+#47 = IFCALIGNMENT2DHORIZONTALSEGMENT($, $, $, #48);
+#48 = IFCLINESEGMENT2D(#49, 0., 400.);
+#49 = IFCCARTESIANPOINT((0., 0.));
+#50 = IFCALIGNMENT2DHORIZONTALSEGMENT($, $, $, #51);
+#51 = IFCTRANSITIONCURVESEGMENT2D(#52, 0., 150., $, 500., .F., .F., .CLOTHOIDCURVE.);
+#52 = IFCCARTESIANPOINT((400., 0.));
+#53 = IFCALIGNMENT2DHORIZONTALSEGMENT($, $, $, #54);
+#54 = IFCCIRCULARARCSEGMENT2D(#55, 351.405633073038, 400., 500., .F.);
+#55 = IFCCARTESIANPOINT((549.662851380011, -7.48795505445));
+#56 = IFCBUILDINGELEMENTPROXY('0s45TNlS97LgwSzWy054YQ', #1, 'subballast example', $, $, #57, #61, $, $);
+#57 = IFCLOCALPLACEMENT(#36, #60);
+#58 = IFCDIRECTION((0., 0., 1.));
+#59 = IFCDIRECTION((1., 0., 0.));
+#60 = IFCAXIS2PLACEMENT3D(#18, #58, #59);
+#61 = IFCPRODUCTDEFINITIONSHAPE($, $, (#67));
+#62 = IFCFIXEDREFERENCESWEPTAREASOLID(#21, #65, #45, $, $, #66);
+#63 = IFCDIRECTION((0., 0., 1.));
+#64 = IFCDIRECTION((1., 0., 0.));
+#65 = IFCAXIS2PLACEMENT3D(#18, #63, #64);
+#66 = IFCDIRECTION((1., 0., 0.));
+#67 = IFCSHAPEREPRESENTATION(#20, 'Body', 'SweptSolid', (#62));
+#68 = IFCBUILDINGELEMENTPROXY('2g$2U96f58Jw0qMUBrAOmy', #1, 'ballast example', $, $, #69, #73, $, $);
+#69 = IFCLOCALPLACEMENT(#36, #72);
+#70 = IFCDIRECTION((0., 0., 1.));
+#71 = IFCDIRECTION((1., 0., 0.));
+#72 = IFCAXIS2PLACEMENT3D(#18, #70, #71);
+#73 = IFCPRODUCTDEFINITIONSHAPE($, $, (#79));
+#74 = IFCFIXEDREFERENCESWEPTAREASOLID(#27, #77, #45, $, $, #78);

```

Comment:
I feel that using ÌfcFixedReferenceSweptAreaSolid` is not the correct choice here. It should rather be the `IfcSectionedSolidHorizontal`.  @TLiebich - opinion?

Reply:
usually sleepers in a railway would follow the local z-direction (perpendicular to the vertical alignment) - so a fixed reference swept area solid would work (not sure about slabtrack, but I would assume that it has a constant thickness independently of the slope). 

but then, the name of the UC should change.